egs/grid/s5/path.sh

```diff
@@ -0,0 +1,15 @@
+# Default working directory for storing data/ exp/ mfcc/ etc
```

Comment:
please use the default path.sh (KALDI_ROOT should be set differently and PATH as well)

as for the REC_ROOT, WAV_ROOT -- I'd suggest putting them into the main script, if viable.